modules/microshift-backing-up-manually.adoc

```diff
@@ -0,0 +1,31 @@
+//Module included in the following assemblies:
+//
+// * microshift_updating/microshift-update-options.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-backing-up-manually_{context}"]
+= Backing up {product-title} data manually
+
+If you are not using an OSTree system, you can back up data manually at any time. Data backups are advised prior to any system updates. Backups are created in the `/var/lib/microshift-backups` directory by default. You can also specify a directory by running an optional backup command.
```

Comment:
> at any time

Sort of, with a caveat: `microshift.service` needs to be stopped before attempting to backup (`# systemctl stop microshift.service`).
But I see it's included in Prerequisites below, so I don't know if we need to be more precise in this line

> You can also specify a directory by running an optional backup command.

(nit on _command_ vs _option_)
From my technical POV which I'm not sure how translate to customer doc but i'll mention it:
- You always need to run a backup command (this is the `backup` in `microshift backup`)
- To specify different storage dir (parent dir of backup dir), you use option `--storage`.